packages/liferay-npm-scripts/package.json

```diff
@@ -65,6 +66,7 @@
 		"prettier": "2.0.5",
 		"react": "*",
 		"react-dom": "*",
+		"react-test-renderer": "*",
```

Comment:
Seeing this is a Facebook-provided library, I've gone with the loose version range like we do for `react` and `react-dom` themselves.